packages/@okta/vuepress-site/docs/concepts/policies/index.md

```diff
@@ -4,116 +4,151 @@ meta:
 
-As a best practice, restrictive rules should be placed at the top of the priority list. Beyond that, you can create combinations of conditions for multiple scenarios. There is no limit to the number of rules that your policies can have.
+See [Configure the password authenticator](https://help.okta.com/oie/en-us/content/topics/identity-engine/authenticators/configure-password.htm) for more information on passwords as an authenticator factor.
 
-Different policy types control settings for different operations. All policy types share a common framework, message structure and API, but have different policy settings and rule data. When a policy is retrieved, for example when the user attempts to sign in to Okta, then policy evaluation takes place:
+### Enrollment policies
 
-* Each policy of the appropriate type is considered in the order that the policies appear in the policy list.
-* Each condition associated with the policy is evaluated:
-  * If one or more conditions can't be met, then the next policy in the list is considered.
-  * If the conditions can be met, then each rule associated with the policy is considered in the order that the rules appear in the rules list.
-* Each condition associated with a given rule is evaluated:
-  * If all of the conditions associated with a rule are met, then the settings contained in the rule and in the associated policy are applied to the user.
-  * If none of the policy rules have conditions that can be met, then the next policy in the list is considered.
+Enrollment policies determine how users enroll an authenticator and what attributes to collect when validating users.
 
-> **Note:** Policies that have no rules aren't considered during evaluation and are never applied.
+#### Authenticator enrollment policies
+
+An [authenticator enrollment policy](/docs/reference/api/policy/#authenticator-enrollment-policy) controls how users enroll an authenticator. The policy controls which of the multifactor authentication (MFA) [methods](https://help.okta.com/okta_help.htm?type=oie&id=ext-about-authenticators) are available for a user and when a user can enroll in a particular factor.
+
+Enable factors in your Okta org by creating a policy with one or more authenticators, and then assigning that policy to your app. See [Authenticators](/docs/guides/authenticators-overview/main/) to learn how to increase the security of your app by requiring a user to verify their identity in more than one way.
+
+> **Note:** In Identity Engine, the Multifactor (MFA) Enrollment Policy name has changed to [authenticator enrollment policy](/docs/reference/api/policy/#authenticator-enrollment-policy).
+
+See [Authenticator enrollment policy](https://help.okta.com/oie/en-us/content/topics/identity-engine/policies/about-mfa-enrollment-policies.htm) for additional information on MFA authentication, enrollment policies, and rules.
```

Comment:
(https://help.okta.com/oie/en-us/content/topics/identity-engine/policies/about-mfa-enrollment-policies.htm) needs to be an alias

```suggestion
See [Authenticator enrollment policy](https://help.okta.com/oie/en-us/content/topics/identity-engine/policies/about-mfa-enrollment-policies.htm) for additional information on MFA authentication, enrollment policies, and rules.
```